app/src/androidTest/java/mozilla/lockbox/uiTests/Navigator.kt

```diff
@@ -153,11 +152,9 @@ class Navigator {
         if (goManually) {
             bypassOnboarding()
         } else {
+            // Go to ItemList after unlocking the device
+            Dispatcher.shared.dispatch(RouteAction.LockScreen)
```

Comment:
Was this the solution? How does this fix the problem?

Reply:
@jhugman Yes, this fixed the problem. Now the app starts in welcome screen, it goes to lock state and it can be unlocked and user is taken to Item List automatically (no need to tap on each Onboarding button). This is the only way I have found to go from Welcome screen to Item list from the tests.